src/group_theory/subgroup.lean

```diff
@@ -1186,6 +1194,88 @@ end
 lemma map_eq_bot_iff_of_injective {f : G →* N} (hf : function.injective f) : H.map f = ⊥ ↔ H = ⊥ :=
 by rw [map_eq_bot_iff, f.ker_eq_bot_iff.mpr hf, le_bot_iff]
 
+end subgroup
+
+namespace subgroup
+
+open monoid_hom
+
+variables {H : Type*} [group H] {f : G →* H}
```

Comment:
Isn't `f` an explicit argument for the `linear_map` versions?

Reply:
Sure, I can put `f` in every lemma. The way it is written now it follows the same explicit/implicit conventions as in `linear_map`, I believe.